README.md

```diff
@@ -135,6 +135,10 @@ fa fa-calendar
 ```
 ![Font awesome](https://raw.githubusercontent.com/alner/qsStatisticBlock/screenshots/screenshots/fontawesome.png)
 
+## Data limits
+Measurements: 1-256
```

Comment:
Nice that you found the limitations in `definition.js` and documented them!

Reply:
thanks :)